tests/datamodules/test_digital_typhoon.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this file needed for coverage or can we delete it and rely on the trainer tests? I've been trying to slowly get rid of `tests/datamodules` since it doesn't actually do a good job of ensuring that the datamodules in fact work.

Reply:
I just added it to test the splitting mechanism explicitly.